pkg/configmap/configmap_watcher.go

```diff
@@ -79,6 +81,11 @@ func (cmw *ConfigMapWatcher) StartInformer(ctx context.Context) error {
 		UpdateFunc: func(oldObj, newObj interface{}) {
 			newCm := newObj.(*apicorev1.ConfigMap)
 
+			// This is sometimes triggered even if no change was due to the ConfigMap
+			if configMapName != newCm.ObjectMeta.Name || reflect.DeepEqual(oldObj, newObj) {
```

Comment:
Yes I think the informer is resynced periodically....is it an issue?

Reply:
Apparently this is expected behaviour, so I've added the reflect library to compare. After the first time it comes in with both old and new object being the same, it causes a return and this avoids further Informer updates with the same change.

But I wanted to "fix" a constant recurrent fake update (because there is no new information) to be updating Validation Engine.